language-server/modules/langserver-core/src/test/resources/completion/function/source/commonCompletionBlocks.bal

```diff
@@ -9,7 +9,7 @@ endpoint jdbc:Client testDB {
     poolOptions: { maximumPoolSize: 5 }
 };
 
-function main(string... args) {
+public function main(string... args) {
```

Comment:
@NatashaWso2 we are this common template for the testing since we need to check whether the completion computations respects the args variable. So, shall we keep this?